### PR TITLE
B fix/request service tests

### PR DESCRIPTION
**Pull request by OliveraBoskov**
Removed unnecessary code, fixed tests.

### CODE REVIEW

File: src/main/java/eu/execom/hawaii/service/SendNotificationsService.java [hunk 1]
```diff
@@ -120,14 +120,14 @@ private void send(String convertedToJson) {
       conn.setRequestProperty("Authorization", "key=" + authorizationKey);
       conn.connect();
 
+      OutputStream os = conn.getOutputStream();
+      OutputStreamWriter osw = new OutputStreamWriter(os, StandardCharsets.UTF_8);
+      osw.write(convertedToJson);
+      osw.flush();
```
Comment (stanislavgrujic) [inline src/main/java/eu/execom/hawaii/service/SendNotificationsService.java]: Why did you remove try clause? Are you aware of the implications?

Comment (OliveraBoskov) [reply src/main/java/eu/execom/hawaii/service/SendNotificationsService.java]: By mistake. Previously there was an error because I changed the order of code and when correcting it I forgot to put try clause. I am aware that I need try clause to close the connection.
